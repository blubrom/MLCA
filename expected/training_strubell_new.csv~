model,base,base,base,changed_density,changed_density,changed_density,changed_location,changed_location,changed_location
,GWP,ADP,PE,GWP,ADP,PE,GWP,ADP,PE
,(\COtwo{}),(\Sbe{}),(MJ),(\COtwo{}),(\Sbe{}),(MJ),(\COtwo{}),(\Sbe{}),(MJ)
Transformer_base,,,,,,,,,
Transformer_big,,,,,,,,,
BERT_base,,,,,,,,,
ELMo,,,,,,,,,