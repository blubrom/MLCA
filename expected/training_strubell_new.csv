model,base,base,base,base,base,base,base,base,base,base,base,base,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedDensity,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation,changedLocation
,GWP,GWP,GWP,GWP,ADP,ADP,ADP,ADP,PE,PE,PE,PE,GWP,GWP,GWP,GWP,ADP,ADP,ADP,ADP,PE,PE,PE,PE,GWP,GWP,GWP,GWP,ADP,ADP,ADP,ADP,PE,PE,PE,PE
,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max,base,min,facebook,max
,(\COtwo{}),(\COtwo{}),(\COtwo{}),(\COtwo{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(MJ),(MJ),(MJ),(MJ),(\COtwo{}),(\COtwo{}),(\COtwo{}),(\COtwo{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(MJ),(MJ),(MJ),(MJ),(\COtwo{}),(\COtwo{}),(\COtwo{}),(\COtwo{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(\Sbe{}),(MJ),(MJ),(MJ),(MJ)
Transformer_base,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
Transformer_big,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
BERT_base,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
ELMo,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
